### PR TITLE
chore(test): Add more node versions for testing

### PR DESCRIPTION
**Pull request by mindmelting**
Ability to test multiple node versions.

@dwmkerr As per other PR this will require new Docker images to be pushed

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -17,12 +17,34 @@ jobs:
...
+    <<: *test-common
+    docker:
+      - image: dwmkerr/node:12-imagemagick
 
 workflows:
   version: 2
   build:
     jobs:
-      - test:
+      - test-node8:
```
Comment (dwmkerr) [inline .circleci/config.yml:37]: Will these run in parallel out of curiosity?

Comment (mindmelting) [reply .circleci/config.yml:37]: Yes they will - from memory there is config to determine dependencies between jobs

Comment (dwmkerr) [reply .circleci/config.yml:37]: Thanks, just noticed the same thing.

**Comment by dwmkerr** [comment]
@mindmelting adding https://github.com/dwmkerr/app-icon/pull/99 as well until we drop node6.

**Pull request merged**